### ISSUE TITLE
Plugin fails with null reference exception if filestorage is not defined

### ISSUE THREAD

**Issue by rowong** [jenkinsci/windows-azure-storage-plugin#79]
After upgrading to 3.6, we've been seeing the following null reference exception.  It looks like the code is trying to always tokenize the fileShareName even if it is not set which results in a null reference exception.

We've downgraded to 3.5 to avoid the issue

```
Archiving artifacts
ERROR: Build step failed with exception
java.lang.NullPointerException
	at com.microsoftopentechnologies.windowsazurestorage.WAStoragePublisher.replaceMacro(WAStoragePublisher.java:373)
	at com.microsoftopentechnologies.windowsazurestorage.WAStoragePublisher.perform(WAStoragePublisher.java:395)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:81)
	at hudson.tasks.BuildStepMonitor$2.perform(BuildStepMonitor.java:32)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:735)
	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:676)
	at hudson.model.Build$BuildExecution.post2(Build.java:186)
	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:621)
	at hudson.model.Run.execute(Run.java:1760)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:405)
Build step 'Upload artifacts to Microsoft Azure Storage' marked build as failure
```

**Comment by ArchangelSDY** [comment]
Sorry for the inconvenience. This has already been fixed in #73 . We will release a new version soon.

**Comment by rowong** [comment]
Can you pull the broken version then so people don't accidently upgrade to it and break themselves?
